README.md

```diff
@@ -62,7 +68,7 @@ This image respects the following custom environment variables to be passed duri
 - `build_time_zones`: `True` builds the timezone db which is needed for time-dependent routing. `Force` will do the same, but first delete the existing db. Default `False`.
 - `build_transit`: `True` will attempt to build transit tiles if none exist yet. `Force` will remove existing transit **and** routing tiles. Default `False`.
```

Comment:
Having both `True` and `False` as a default might be somewhat confusing.

Would you think of a PR changing this to a table like the following as a good contribution?

| Environment Variable | Default | Description |
|--------|--------|--------|
| `build_transit` | `False` | `True` will attempt to build transit tiles if none exist yet<br> `Force` will remove existing transit **and** routing tiles |